### DISCUSSION TITLE
get Language option to repository

### DISCUSSION THREAD

**Discussion by Asbaharoon** [community/community#30768] [Other Feature Feedback, Questions, & Ideas]
![get to show 2 option](https://user-images.githubusercontent.com/98842320/186639338-60783325-626e-4db3-85ab-15a4ac4fb2a3.png)
Sorry my friend 

i edit my request and i need to get 2 options as my clarified by red color, for you help please 

thank you and best regards,

**Comment by Redume** [comment]
In the sense of getting? How and where? If I understood correctly, this can be done through the Github Api

[Example](https://api.github.com/repos/github/fetch)

**Reply by Asbaharoon** [reply]
ok thank you and i will check it later

**Comment by Asbaharoon** [comment]
in the right side of repository as i clarified by red color

احصل على Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
